src/material-experimental/mdc-slider/slider.spec.ts

```diff
@@ -1,1468 +1,7 @@
-import {BidiModule} from '@angular/cdk/bidi';
```

Comment:
Rather than deleting the tests, would it make more sense to leave them here and incrementally get them passing?

Reply:
The short answer is no, I don't think it would. All of these tests would need to be adapted to the new api, and many of them either no longer make sense to test - like tests that confirm the css fill & transforms are set correctly. I'm still planning on using these tests as a reference for the new ones I'll be adding in later, but I'll need to go through them one by one to see if they're still relevant.